src/components/AccommodationCard.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  CardActionArea,
+  Stack,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import Star from "../assets/images/Vector.svg";
+
+type AccommodationCardProps = {
+  title: string;
+  location: string;
+  price: number;
+  categorization: number;
+};
+
+export const AccommodationCard = ({
+  title,
+  location,
+  price,
+  categorization,
+}: AccommodationCardProps): JSX.Element => {
+  const { colors } = useTheme();
+  const stars = {
+    backgroundImage: `url(${Star})`,
+    width: "20px",
+    height: "19px",
+  };
+
+  return (
+    <Card sx={{ maxWidth: "18.5625rem", boxShadow: "0" }}>
+      <CardActionArea>
+        <CardMedia
+          sx={{
+            borderRadius: "12px",
+            width: "18.5625rem",
+            height: "16.625rem",
+          }}
+          component="img"
+          image="/accommodation-card.png"
+          alt=""
+        />
+        <CardContent>
+          <Typography
+            component="div"
+            sx={{
+              fontFamily: "Roboto",
+              fontWeight: "500",
+              fontSize: "20px",
+              color: colors.textBlack,
+              marginBottom: "0.5rem",
+            }}
+          >
+            {title}
+          </Typography>
+          <Typography
+            component="div"
+            sx={{
+              fontFamily: "Roboto",
+              fontWeight: "400",
+              fontSize: "15px",
+              color: colors.cardGray,
+              marginBottom: "0.5rem",
+            }}
+          >
+            {location}
+          </Typography>
+          <Typography
+            component="div"
+            sx={{
+              fontFamily: "Roboto",
+              fontWeight: "500",
+              fontSize: "15px",
+              color: colors.textMenuItems,
+              marginBottom: "0.5rem",
+            }}
+          >
+            EUR {price}
+          </Typography>
+          <Stack direction="row" spacing={1}>
+            <div style={stars}></div>
```

Comment:
Instead of using a div with a style of the star, maybe you could try making the Star svg as a react component so it's Star.tsx returning the svg and then calling it here

Reply:
Even better solution that came to my mind now is to create a `Rating` component that takes a number of stars you wish to show and takes care of all of display the desired amount of stars. You don't have to do this now but would be a better solution here in the future but also in general